### ISSUE TITLE
Network error

### ISSUE THREAD

**Issue by guushoekman** [Anthony-Reboul/AndroidDeepL#53]
I think this is very similar to issue #41 and #50. Anytime I try to translate something, a network error is thrown. Retrying doesn't work.

![https://i.imgur.com/bwWiTsF.jpg](https://i.imgur.com/wtuQyAs.jpg)

**Comment by Anthony-Reboul** [comment]
Yes I can confirm, same as #50 . I'll try to release the fix asap.

**Comment by guushoekman** [comment]
Thanks! May I ask why you closed the issue though? The problem still remains.

The reason I opened a new issue is because #50 had been closed. I thought that had been fixed, but that the same problem had once again appeared because of an external change.

If someone now downloads your app and gets a network error, they might go to this repo and check the issues. None of the open issues are related to this, so logically they will open another issue.

Doesn't it make sense to keep either this one or #50 open until you have actually released the fix?
